src/node/server/serverPluginHmr.ts

```diff
@@ -308,9 +316,9 @@ export const hmrPlugin: ServerPlugin = ({
               `${vueBoundary} reloaded due to change in ${relativeFile}.`
           )
           send({
-            type: 'vue-reload',
+            type: 'js-update',
             path: vueBoundary,
-            changeSrcPath: publicPath,
+            changeSrcPath: vueBoundary,
             timestamp
           })
```

Comment:
Here change because we hope re-import vue module when some deps update.eg. js.json.module.css.
Well.The better waye is codegen for this deps by hmr api.like 
``` ts
hot.accpet('a.vue','module.css', () => {
  import('a.vue')
})

```